openpype/settings/entities/schemas/system_schema/host_settings/schema_fusion.json

```diff
@@ -19,6 +19,36 @@
             "label": "Environment",
             "type": "raw-json"
         },
+        {
+            "type": "splitter"
+        },
+        {
+            "type": "dict",
```

Comment:
Do you think move it to the project setting only or leave it in system too? So if there's no per project project override, system would be used?

Reply:
> Do you think move it to the project setting only or leave it in system too? So if there's no per project project override, system would be used?

The project tab have settings that gets used by default by all projects if not changed so that would apply on all projects automatically. I don't see any need to have it in the system tabs too.